### DISCUSSION TITLE
LGT8FX8P ISP programmer using FT232R

### DISCUSSION THREAD

**Discussion by malaouar** [dbuezas/lgt8fx#308] [Ideas]
Now I can burn flat binary programs into the LGT8F328P using an FT232R in bitbang mode.
HEX files need some parsing, I'm stil reading Intel hex format.
Using a test program:
The PB5 LED blinks as expected.
The UART works as a charm.
But I've got a weird text when printing some regs contents in hex format!!! 
![FT232R](https://github.com/dbuezas/lgt8fx/assets/69061342/1a33e589-4304-4530-b54a-b33eca48877d)

The same test program when flashed with the bootloader or the LGTISP it works as expected.
![LGTISP](https://github.com/dbuezas/lgt8fx/assets/69061342/d61d75f2-b5e3-4977-b2e7-d9fa2a94da63)

Here's the FULL code:
https://github.com/malaouar/FT232R_LGTISP

**Comment by dbuezas** [comment]
<img width="190" alt="image" src="https://github.com/dbuezas/lgt8fx/assets/777196/800d78df-7790-4d2f-806a-40b9695aa157">

Or just copy and paste the screenshot inside the text box

**Reply by malaouar** [reply]
Ok, Thanks David.

**Comment by malaouar** [comment]
When verifying the contents of the flash after programming I get some different bytes (usually 0xFF) each time I do a read!!
So surely  the FT232 interface changes or loses some bytes during write/read operations.
